Sources/ESRefreshComponent.swift

```diff
@@ -115,97 +125,97 @@ open class ESRefreshComponent: UIView {
             }
         }
     }
-    
-}
-
-extension ESRefreshComponent /* KVO methods */ {
-    
-    fileprivate static var context = "ESRefreshKVOContext"
-    fileprivate static let offsetKeyPath = "contentOffset"
-    fileprivate static let contentSizeKeyPath = "contentSize"
-    
-    public func ignoreObserver(_ ignore: Bool = false) {
-        if let scrollView = scrollView {
-            scrollView.isScrollEnabled = !ignore
-        }
-        isIgnoreObserving = ignore
-    }
-    
-    fileprivate func addObserver(_ view: UIView?) {
-        if let scrollView = view as? UIScrollView, !isObservingScrollView {
-            scrollView.addObserver(self, forKeyPath: ESRefreshComponent.offsetKeyPath, options: [.initial, .new], context: &ESRefreshComponent.context)
-            scrollView.addObserver(self, forKeyPath: ESRefreshComponent.contentSizeKeyPath, options: [.initial, .new], context: &ESRefreshComponent.context)
-            isObservingScrollView = true
-        }
-    }
-    
-    fileprivate func removeObserver() {
-        if let scrollView = superview as? UIScrollView, isObservingScrollView {
-            scrollView.removeObserver(self, forKeyPath: ESRefreshComponent.offsetKeyPath, context: &ESRefreshComponent.context)
-            scrollView.removeObserver(self, forKeyPath: ESRefreshComponent.contentSizeKeyPath, context: &ESRefreshComponent.context)
-            isObservingScrollView = false
-        }
-    }
-    
-    override open func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey : Any]?, context: UnsafeMutableRawPointer?) {
-        if context == &ESRefreshComponent.context {
-            guard isUserInteractionEnabled == true && isHidden == false else {
-                return
-            }
-            if keyPath == ESRefreshComponent.contentSizeKeyPath {
-                if isIgnoreObserving == false {
-                    sizeChangeAction(object: object as AnyObject?, change: change)
-                }
-            } else if keyPath == ESRefreshComponent.offsetKeyPath {
-                if isIgnoreObserving == false {
-                    offsetChangeAction(object: object as AnyObject?, change: change)
-                }
-            }
-        } else {
-
-        }
-    }
-    
-}
 
-public extension ESRefreshComponent /* Action */ {
+    // MARK: - Actions
 
     public final func startRefreshing(isAuto: Bool = false) -> Void {
         guard isRefreshing == false && isAutoRefreshing == false else {
             return
         }
-        
+
```

Comment:
Since we want to push this back to the original pod, we might want to consider removing any whitespace changes to clean up the diff

Reply:
So since our fork is behind their `master`, I actually just opened a separate PR: https://github.com/eggswift/pull-to-refresh/pull/142